wordpress-mobile/WordPress-FluxC-Android
# Clean up after post integreation tests

Adds "clean up" phase to Post integration tests.

To test:
1. Check the number of Published/Draft/Scheduled/Trashed posts on your test site 
2. Run the tests
3. Verify the number of posts hasn't changed.
